bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala

```diff
@@ -19,7 +19,9 @@ import scalaz.Scalaz._
  * Unique configurations represent sets of observes in a sequence which are done with the same
  * instrument configuration and exposure times.
  */
-case class ItcUniqueConfig(count: Int, labels: String, config: Config) {
+case class ItcUniqueConfig(count: Int, labels: String, configs: Seq[Config]) {
+
+  def config = configs.head
```

Comment:
I imagine you only create these when you have a `configs` sequence with values but a `NonEmptyList` might  be a good idea here to have the compiler enforce that.